source/MaterialXView/Viewer.h

```diff
@@ -95,6 +95,42 @@ class Viewer : public ng::Screen
         _modifiers = modifiers;
     }
 
+    // Set the bake width of the image.
+    void setBakeWidth(const int bakeWidth)
```

Comment:
The "const" modifier isn't needed for this integer argument, since it's passed by value.